pkg/kubelet/pleg/generic.go

```diff
@@ -341,6 +327,9 @@ func (g *GenericPLEG) Relist() {
 			case g.eventChannel <- events[i]:
 			default:
 				metrics.PLEGDiscardEvents.Inc()
+				// When the event channel is full,
+				// We'll continue to request pod resync to prevent event loss.
+				g.RequestPodReSync(pid, true)
```

Comment:
If there are multiple events added to `events` in this iteration of the loop for this pod, couldn't this cause unnecessary pod resyncs? 

I think we want to be careful here, we only need to re-request a pod sync if we haven't already emitted another event that does so